CHANGELOG.md

```diff
@@ -18,6 +18,7 @@
 ### ⬆️ Improved
 
 ### ✅ Added
+- Added `isFilteringMessages` check on request. [#3948](https://github.com/GetStream/stream-chat-android/pull/3948)
```

Comment:
What request specifically? Maybe being a bit more descriptive here if necessary could be good.